framework/src/controller/application.js

```diff
@@ -46,6 +46,68 @@ const registerProcessHooks = app => {
 	process.once('exit', (error, code) => app.shutdown(code, error));
 };
 
+/* As for now we don't support this constants to be configured
+by the user. This will change in the future */
+const nonConfigurableConstants = {
```

Comment:
maybe put this value in `/controller/constants.js`?

Reply:
There is also `/controller/schema` constants which also contains default values... Wouldn't creating another file named constants also inside `/controller` be confusing?

Reply:
Just discovered doing it however you say simplifies loading those constants in unit tests setup. This is actually why now they're failing